### ISSUE TITLE
az containerapp logs show KeyError: 'logStreamEndpoint'

### ISSUE THREAD

**Issue by ushuz** [Azure/azure-cli-extensions#5900]
## Describe the bug

**Command Name**
`az containerapp logs show`
Extension Name: containerapp. Version: 0.3.9.

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'logStreamEndpoint'
Traceback (most recent call last):
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/knack/cli.py", line 233, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 363, in handler
    show_exception_handler(ex)
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/arm.py", line 429, in show_exception_handler
    raise ex
  File "/usr/local/Cellar/azure-cli/2.45.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 361, in handler
    return op(**command_args)
  File "/Users/ushuz/.azure/cliextensions/containerapp/azext_containerapp/custom.py", line 2274, in stream_containerapp_logs
    logstream_endpoint = token_response["properties"]["logStreamEndpoint"]
KeyError: 'logStreamEndpoint'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az containerapp logs show -n {} --tail {}`

## Expected Behavior

To stream containerapp logss

## Environment Summary
```
macOS-12.6.3-x86_64-i386-64bit, Darwin 21.6.0
Python 3.10.10
Installer: HOMEBREW

azure-cli 2.45.0

Extensions:
containerapp 0.3.9

Dependencies:
msal 1.20.0
azure-mgmt-resource 21.1.0b1
```

**Comment by ushuz** [comment]
Never mind, fixed by upgrading containerapp extension to 0.3.21

**Comment by navba-MSFT** [comment]
@ushuz Thanks for the update. If you need any further assistance on this issue in future, please feel free to reopen this thread. We would be happy to help.
